password/password.go

```diff
@@ -50,10 +51,20 @@ func Prompt() (string, error) {
 // we aren't using a terminal for standard input
 func readPassFromStdin() (string, error) {
 	pass := []byte{}
+	var err error
+	fi, _ := os.Stdin.Stat()
+
+	if (fi.Mode() & os.ModeCharDevice) == 0 {
+		log.Logv(log.DebugLow, "reading data from stdin pipe")
```

Comment:
Not sure if this is the right log level.

Reply:
Oh, good point. I'll make those adjustments!

Also, I was planning on tagging reviewers once Huan merged his PR! I'll add Tim after I re-request your review.